doc/cookbook/source/examples/classifier/hierarchical.rst

```diff
@@ -0,0 +1,36 @@
+=======================
+Hierarchical Clustering
+=======================
+
+Hierarchical clustering is a method of cluster analysis which seeks to build a hierarchy of clusters.
+In Shogun, we applies "bottom up" approach for the hierarchical clustering: each observation starts in its own cluster, and pairs of clusters are merged as one moves up the hierarchy.
```

Comment:
We apply a "bottom up" approach: each observation starts in its own clister, and pairs of clusters are subsequently merged.

Cleaner language